packages/graphql/tests/tck/issues/missing-custom-cypher-on-unions.ts

```diff
@@ -0,0 +1,236 @@
+            @limit(default: 100, max: 300) {
+            iri: ID! @id @alias(property: "uri")
+            relatesToChild: [hierarchicalNodeTarget!]!
+                @relationship(type: "relatesToChild", properties: "RelateProps", direction: OUT)
+            isContained: HierarchicalRoot! @relationship(type: "isContained", properties: "RelateProps", direction: OUT)
+
+            hierarchicalPathNodes: [choNode]
+                @cypher(
+                    statement: """
+                    MATCH p=(this)<-[:relatesToChild*..10]-(parent:HierarchicalRoot)
+                    WITH p, parent
+                    OPTIONAL MATCH (parent) - [:type] -> (parentType)
+                    WITH p, parent, parentType
+                    UNWIND nodes(p) as pathNode
+                    WITH pathNode, relationships(p) as rels, parent, parentType
+                    WITH pathNode, head([r in rels where endNode(r) = pathNode]) as relation, parent, parentType
+                    OPTIONAL MATCH (pathNode) - [:identifier] -> (pathNodeIdentifier)
+                    OPTIONAL MATCH (pathNode) - [:title] -> (pathNodeTitle)
+                    WITH pathNode, relation, parent, parentType, pathNodeIdentifier, collect({value: pathNodeTitle.value, language: pathNodeTitle.language}) as pathNodeTitles
+                    WITH {
+                    uri: pathNode.uri,
+                    identifier: pathNodeIdentifier.value,
+                    title: pathNodeTitles,
+                    hasSortKey: relation.hasSortKey,
+                    labels: labels(pathNode),
+                    partOf: parent.uri,
+                    partOfType: parentType.uri
+                    } AS obj
+                    RETURN DISTINCT obj as result
+                    """
+                    columnName: "result"
+                )
+        }
+        type choNode @mutation(operations: []) {
+            iri: ID! @id @alias(property: "uri")
+            identifier: String
+            title: [title]
+            hasSortKey: String
+            labels: [String]
+            partOf: ID
+            partOfType: String
+        }
+
+        type title @mutation(operations: []) {
+            value: String
+            language: String
+        }
+
+        union relatesToChildTarget = Expression | Work | Fragment | HierarchicalRoot
+
+        interface RelateProps @relationshipProperties {
+            hasSortKey: String
+            isIdentifiedBy: ID
+        }
+    `;
+
+    beforeAll(() => {
+        neoSchema = new Neo4jGraphQL({
+            typeDefs,
+        });
+    });
+
+    test("should include checks for auth jwt param is not null", async () => {
+        const query = gql`
+            query browseHierarchicalComponents($hierarchicalRootId: ID!, $choNodeIris: [ID!]!) {
+                hierarchicalComponents(where: { isContained: { iri: $hierarchicalRootId }, iri_IN: $choNodeIris }) {
+                    #...hierarchicalComponentFields
```

Comment:
```suggestion
```

Reply:
This is actually part of the initial bug report, so I think we should leave it

Reply:
Sounds good!